src/js/components/Select/doc.js

```diff
@@ -23,6 +23,27 @@ export const doc = Select => {
       { active, disabled, selected } keys indicating the current state
       of the option.`,
     ),
+    clear: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.shape({
+        position: PropTypes.oneOf(['top', 'bottom'])
+          .description(
+            `Add a clear value to the top or at the bottom of the
+            container. By default no clear option is present.`,
+          )
+          .defaultValue('top'),
+        label: PropTypes.string
+          .description('Label for the clear selection item')
+          .defaultValue('Clear selection'),
+        render: PropTypes.func.description(
```

Comment:
Has someone asked for `render` or is this something we're anticipating? I ask because I'd like to keep the API surface as small as we can until a need arises.

Reply:
I don't think anyone has asked for it, i took it from this pr
#3736

Reply:
removed render